tools/symbolicator/XamGlueConstants.cs

```diff
@@ -298,7 +298,7 @@ internal static class XamGlueConstants {
 		[SymbolicatorInfo (Skip = true)]
 		internal const string EveryProtocol_MetadataAccessor = "$s7XamGlue13EveryProtocolCMa";
 		[SymbolicatorInfo (Skip = true)]
-		internal const string EveryProtocolNew = "$s7XamGlue13EveryProtocolCACycfC";
+		internal const string MakeEveryProtocol = "$s7XamGlue17makeEveryProtocolAA0dE0CyF";
```

Comment:
🤔 Does this string get generated + what is a symbolicator?

Reply:
Symbolicator is something that @dalexsoto did. For me, I pulled the symbol out of XamGlue.dylib